src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -674,19 +713,41 @@ public void blobAclUpdates(
     }
   }
 
+  /**
+   * Non-static version of {@link GcsPdao#getBlobFromGsPath(Storage, String, String)} to enable
```

Comment:
Ah dang, nice to be able to mock this but this smells funny.  Do we need to maintain the existing static version?

Reply:
Seems like mocking `Storage` would be better if it's not too hard to do.

Reply:
Hmm...lemme see if I can do that.  I agree it's less gross if we can avoid the mixed static / non-static thing

Reply:
k, mocked out storage. I like it way better.  thanks for the suggestion!